src/main/java/core/basesyntax/service/impl/ReportGeneratorServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGeneratorService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportGeneratorServiceImpl implements ReportGeneratorService {
+    private StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public String report() {
+        List<String> report = new ArrayList<>();
```

Comment:
```suggestion
        StringBuilder builder = new StringBuilder();
```

Reply:
really no need to create list of strings and then convert it to one big string
you can just do it directly using string builder, completely bypassing list creation